src/Core/Silk.NET.Core/Silk.NET.Core.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=miscellaneous/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
What's this?

Reply:
Stops resharper from complaining about the namespace being wrong.